controllers/RestaurantController.ts

```diff
@@ -11,7 +11,7 @@ import RestaurantControllerI from "../interfaces/RestaurantControllerI";
  * <ul>
  *     <li>GET /api/restaurants to retrieve all the restaurant instances</li>
  *     <li>GET /api/restaurants/:rid to retrieve a particular restaurant instance</li>
- *     <li>POST /api/restaurants to create a new restaurant instance</li>
+ *     <li>POST /api/users/:uid/restaurants to create a new restaurant instance</li>
```

Comment:
I think this should no longer include user/:uid in the path. Tried it out on Postman and based on implementation, creating a restaurant is /api/restaurants

Reply:
Updated. Thank you!